xrpl/models/requests/request.py

```diff
@@ -58,6 +58,9 @@ class RequestMethod(str, Enum):
     # NFT methods
     NFT_BUY_OFFERS = "nft_buy_offers"
     NFT_SELL_OFFERS = "nft_sell_offers"
+    NFT_INFO = "nft_info"
```

Comment:
Should this be documented as clio only methods?

Reply:
added comments